code-challenges.test.js

```diff
@@ -11,42 +11,174 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------1) Create a function that takes in a string and returns a string with a coded message. The coded message converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.
-
+// psuedocode:
+// create function named codedMessage that takes in a "string" and outputs a "string" function must look at each letter individually so i may create an emplty array and push the given variables into it. iterate over the array lookinng at each value in each index filteringor let statement convert vowels a, e, i, o to
+// input:"string"
+// output:"string"
 // a) Create a test with expect statements using the variables provided.
 
-const secretCodeWord1 = "Lackadaisical"
+describe("codedMessage", () => {
+  it("takes in a string and returns a string with a coded message.", () => {
+    const secretCodeWord1 = "Lackadaisical";
+    // Expected output: "L4ck4d41s1c4l"
+    const secretCodeWord2 = "Gobbledygook";
+    // Expected output: "G0bbl3dyg00k"
+    const secretCodeWord3 = "Eccentric";
+    // Expected output: "3cc3ntr1c"
+
+    expect(codedMessage(secretCodeWord1)).toEqual(["L4ck4d41s1c4l"]);
+    expect(codedMessage(secretCodeWord2)).toEqual(["G0bbl3dyg00k"]);
+    expect(codedMessage(secretCodeWord3)).toEqual(["3cc3ntr1c"]);
+  });
+});
+
+// ---> FAIL  ./code-challenges.test.js
+//   codedMessage
+//   ✕ takes in a string and returns a string with a coded message. (1 ms)
+// ---> ReferenceError: codedMessage is not defined
+
+// b) Create the function that makes the test pass.
+
+// Create a function that takes in a string and returns a string with a coded message. The coded message converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.
+// if(lowerCaseString.length.incluedes("a", "e", "i", "o")){
+
+const secretCodeWord1 = "Lackadaisical";
 // Expected output: "L4ck4d41s1c4l"
-const secretCodeWord2 = "Gobbledygook"
+const secretCodeWord2 = "Gobbledygook";
 // Expected output: "G0bbl3dyg00k"
-const secretCodeWord3 = "Eccentric"
+const secretCodeWord3 = "Eccentric";
 // Expected output: "3cc3ntr1c"
 
-// b) Create the function that makes the test pass.
+const codedMessage = (string) => {
+  for (let i = 0; i < string.length; i++) {
+    const lowerCaseString = string.toLowerCase();
+    {
+      if (lowerCaseString.includes("a", "e", "i", "o")) set: a = "4";
+      set: e = "3";
+      set: i = "1";
+      set: o = "0";
+    }
+  }
+  return lowerCaseString;
+};
```

Comment:
I see what you're trying to do, but I believe you are confusing yourself with things you are finding in your research.
1) The `.includes` method expects a single argument, so calling it with 4 arguments like you are here is not valid syntax.
2) The curly braces on lines 55 & 60 are unnecessary in JavaScript
3) The `set` statements are not valid JavaScript syntax
I would suggest you look into what JavaScript built in methods you can use to perform what you are wanting to do. Break the problem down step by step and find a solution for each step before moving on to the next one.